src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -181,7 +209,7 @@ private static void SetEnvVariables()
         /// </summary>
         /// <param name="reps">Replacement strings.</param>
         /// <returns>Json string with replacements.</returns>
-        public static string GetModifiedJsonString(string[] reps)
+        public static string GetModifiedJsonString(string[] reps, string enumString)
```

Comment:
nit: comment needed for `enumString`. From the test's code, it seems this value is used to provide an actual value for 'database-type' regardless of whether `replaceEnvVar` is true or false?

Reply:
Thats right, fixed. Yes its used to provide actual value when called for the expectedConfig. For actual config, this would be `@env('enumVarName')`